go/updater/updater_windows.go

```diff
@@ -0,0 +1,48 @@
+// Copyright 2015 Keybase, Inc. All rights reserved. Use of
+// this source code is governed by the included BSD license.
+
+// +build windows
+
+package updater
+
+import (
+	"fmt"
+	"os/exec"
+	"path/filepath"
+	"strings"
+
+	keybase1 "github.com/keybase/client/go/protocol"
+)
+
+func (u *Updater) applyUpdate(ui UI, update keybase1.Update) (tmpPath string, err error) {
+	if err = u.promptForUpdateAction(ui, update); err != nil {
+		return
+	}
+
+	if update.Asset.LocalPath == "" {
+		err = fmt.Errorf("No local asset path for update")
+		return
+	}
+
+	// The default source will be an .exe on Windows
+	// ...but still do this for .zip files, at least so tests will pass
+	if strings.HasSuffix(update.Asset.LocalPath, ".zip") {
```

Comment:
I wouldn't keep this code just to make the test pass... We can make platform specific tests?